le_dash/banner.py

```diff
@@ -1,18 +1,37 @@
 import six
+import hashlib
 import requests
 from django.conf import settings
+from django.core.cache import cache
+import logging
 
 if six.PY2:
     from urlparse import urljoin
 else:
     from urllib.parse import urljoin
 
 
+logger = logging.getLogger(__name__)
+
+
 def banner_req(url, params):
+    #  md5 is cheaper than urlunparse/urlencode params
+    get_url = hashlib.md5("{0}".format(params))
```

Comment:
`get_url` is being assigned the md5 hash object. to get the cache key string you need to call, e.g., `hexdigest()`.

The url is always the same *now* but the first time someone wants to add a new type of request for banner data they're going to have to refactor the caching too. also not sure why you'd need to urlencode/urlunparse. 

here's what I would expect:

    cache_key = hashlib.md5("{0} {1}".format(url, params).hexdigest()